lib/cfa/placer.rb

```diff
@@ -1,6 +1,20 @@
 module CFA
-  # allows to place element at the end of configuration. Default one.
-  class AppendPlacer
+  # Places a new {AugeasElement} into an {AugeasTree}.
+  # Children of this abstract class implement different ways **where**
+  # to place the entry.
+  class Placer
+    # @param  [AugeasTree] tree
+    # @return [AugeasElement,Hash] the new element; it is empty!
+    #   Note that the return value is actually a Hash; {AugeasElement}
+    #   documents its structure.
+    def new_element(tree)
+      abstract_method(tree)
+    end
+  end
+
+  # Places the new element at the end of the tree.
+  # Other classes use this Placer by default.
```

Comment:
Ah, in original I mean something like the most common one. or trivial, so probably default wasn't right word.